### ISSUE TITLE
Improve documentation

### ISSUE THREAD

**Issue by balazsorban44** [onefinestay/react-daterange-picker#196]
Hi! I am looking at your project, and it really looks amazing, probably would cover all my needs, but I have so hard time to set it up. And I cannot find any documentation on it. Maybe I am not looking well enough? I had to go to the Issues page just to find out that the basic CSS has to be imported as said in #178 
Though I still cannot work out how to do own stylings, or why the basic unavailable etc. stylings are not showing. Please, take the time to make a good docs page, cause this project has a big potential in my opinion!

**Comment by AlanFoster** [comment]
Hi, thanks for raising this ssue. Have you checked out the getting started section that links to a simple example?

https://github.com/onefinestay/react-daterange-picker/blob/master/README.md#getting-started

**Comment by balazsorban44** [comment]
Yes, I can see all the examples, but beside from the very first one on the site, there is no code example! And I cannot even get the first one working. I get this error while setting the `stateDefinitions`:

![image](https://user-images.githubusercontent.com/18369201/41384609-261765dc-6f77-11e8-8868-74998bae81fe.png)


Now, I am making this issue a more specific one, but I still think the project lacks a good documentation, like a table on the `README.md` with the components' all available props, possible types/effects, styling guides, and so on.
